packages/tabs/tab.md

```diff
@@ -7,93 +7,84 @@ An `<sp-tab>` element surfaces a `label` attribute to serve as its default text
 [![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/tabs?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/tabs)
 [![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/tabs?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/tabs)
 
-```
+```bash
 yarn add @spectrum-web-components/tabs
 ```
 
 Import the side effectful registration of `<sp-tab>` via:
 
-```
+```js
 import '@spectrum-web-components/tabs/sp-tab.js';
 ```
 
 When looking to leverage the `Tab` base class as a type and/or for extension purposes, do so via:
 
+```js
+import { Tab } from '@spectrum-web-components/tabs';
 ```
-import {
-    Tab,
-} from '@spectrum-web-components/tabs';
+
+### Anatomy
+
+The `<sp-tab>` element consists of a label and an optional icon.
+
+```html
+<sp-tab label="Label">
+    <sp-icon-checkmark slot="icon"></sp-icon-checkmark>
+</sp-tab>
 ```
 
-## Examples
+### Options
+
+#### Icon only
+
+For tabs that have an icon and no visible label, the icon should have an `aria-label`. Icons should not be used just as decoration. If the tab item does not have a visible label, it must still have a tooltip to disclose the label.
 
 ```html
-<sp-tabs selected="1">
-    <sp-tab label="Tab 1" value="1"></sp-tab>
-    <sp-tab value="2">Tab 2</sp-tab>
-    <sp-tab label="Tab 3" value="3">
-        <sp-icon-checkmark slot="icon"></sp-icon-checkmark>
-    </sp-tab>
-    <sp-tab vertical value="4">
-        Tab 4
-        <sp-icon-checkmark slot="icon"></sp-icon-checkmark>
-    </sp-tab>
-    <sp-tab-panel value="1">
-        Content for Tab 1 which uses an attribute for its content delivery
-    </sp-tab-panel>
-    <sp-tab-panel value="2">
-        Content for Tab 2 which uses its text content directly
-    </sp-tab-panel>
-    <sp-tab-panel value="3">
-        Content for Tab 3 which uses an attribute with a
-        <code>[slot="icon"]</code>
-        child
-    </sp-tab-panel>
-    <sp-tab-panel value="4">
-        Content for Tab 4 which uses both its text content and a
-        <code>[slot="icon"]</code>
-        child displayed using the
-        <code>[vertical]</code>
-        attribute to define their alignment
-    </sp-tab-panel>
+<sp-tab value="complete" aria-label="Tab with checkmark">
+    <sp-icon-checkmark
+        slot="icon"
+        label="Checking your work"
+    ></sp-icon-checkmark>
+</sp-tab>
```

Comment:
For tabs with icon only, you can label the tab using the icon's `label` attribute:

```suggestion
    <sp-tabs>
        <sp-tab value="complete">
            <sp-icon-checkmark
                slot="icon"
                label="Checking your work"
            ></sp-icon-checkmark>
        </sp-tab>
       <sp-tab-panel value="complete">
           A screenreader will read this tab as "Checking your work"
       </sp-tab-panel>
    <sp-tabs>
```